<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsItemPositionDialogBase</class>
 <widget class="QDialog" name="QgsItemPositionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set item position</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="mPositionGroupBox">
     <property name="title">
      <string>Item reference point</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="mUpperLeftCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="mUpperMiddleCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="mUpperRightCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="mMiddleLeftCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="mMiddleCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="mMiddleRightCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="mLowerLeftCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="mLowerMiddleCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="mLowerRightCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="mCoordinatesGroupBox">
     <property name="title">
      <string>Coordinates</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_1">
      <item row="0" column="0">
       <widget class="QLabel" name="mXLabel">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="mXLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mYLabel">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="mYLineEdit"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="mWidthLabel">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="mWidthLineEdit"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="mHeightLabel">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="mHeightLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>201</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="mSetPositionButton">
       <property name="text">
        <string>Set Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCloseButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
